library/src/doo/shell.clj

```diff
@@ -37,13 +37,20 @@
       (.close r)
       (.toString out))))
 
-(defn exec! [cmd]
+(defn -exec
```

Comment:
Not sure on this naming, perhaps exec* instead? It could also be good to inline this function into line 49?